src/strategies/eden-online-override/examples.json

```
@@ -0,0 +1,24 @@
+[
+  {
+    "name": "Eden Online Override",
+    "strategy": {
+      "name": "eden-online-override",
+      "params": {
+        "symbol": "UBINIJI",
+        "address": "0x66efaf92df6456c3cb810012b2de3fb223c25d0d",
+        "decimals": "0",
```

This should be a number

```suggestion
        "decimals": 0,
```

---

Updated this type